helm/templates/deployment.yaml

```diff
@@ -39,8 +39,16 @@ spec:
           volumeMounts:
             - name: keystore-volume
               mountPath: /person-restapi-keystore
+          env: {{ toYaml .Values.env.database | nindent 12 }}
       initContainers:
-        - name: person-database
+        - name: person-schema-creation
+          image: quay.io/ukhomeofficedigital/postgres-client:latest
```

Comment:
this looks like an old image with 6 critical issues
https://quay.io/repository/ukhomeofficedigital/postgres-client?tab=tags
Can we add an image to the sync in the devops repo or build our own

Reply:
Will investigate that one, Stephen.

Reply:
changed image to quay.io/ukhomeofficedigital/postgres:latest which has no vulnerabilities but is a bit 'oldish'
Can we keep that one or shall we build our own? Maybe it will be overkill just to run 1 SQL command?

Reply:
Thanks. According to the report is has no issues and whilst I am dubious about the accuracy the sync option is not available at the moment